maat-court-data-api/src/main/java/gov/uk/courtdata/users/controller/UserSummaryController.java

```diff
@@ -38,4 +41,39 @@ public ResponseEntity<UserSummaryDTO> getUserSummary(
         return ResponseEntity.ok(userSummaryService.getUserSummary(username));
     }
 
+    @GetMapping(value = "/{username}")
+    @Operation(description = "Retrieve a reservation")
+    @ApiResponse(responseCode = "200", content = @Content(mediaType = MediaType.APPLICATION_JSON_VALUE))
+    @StandardApiResponse
+    @NotFoundApiResponse
+    public UserEntity getUser(@PathVariable String username) {
+        return userSummaryService.getUser(username);
+    }
+
+    @PostMapping("/")
+    @Operation(description = "Create a user")
+    @StandardApiResponse
+    @ApiResponse(responseCode = "200", content = @Content(mediaType = MediaType.APPLICATION_JSON_VALUE))
+    public ResponseEntity<Void> createUser(@RequestBody UserEntity userEntity) {
+        userSummaryService.createUser(userEntity);
+        return ResponseEntity.ok().build();
+    }
+
+    @PatchMapping("/{username}")
+    @Operation(description = "Update user details")
+    @StandardApiResponseCodes
```

Comment:
Do we need the `@NotFoundApiResponse` annotation here aswell, as the patch service calls the get service to find the user to update which throws a `RequestedObjectNotFoundException` if one can't be found?

Reply:
Good spot. Added the annotation.